core/src/main/java/jenkins/model/ParameterizedJobMixIn.java

```diff
@@ -138,7 +145,7 @@ public final boolean scheduleBuild(int quietPeriod, Cause c) {
         return Jenkins.getInstance().getQueue().schedule2(asJob(), quietPeriod, queueActions).getItem();
     }
 
-    private List<ParameterValue> getDefaultParametersValues() {
+    public List<ParameterValue> getDefaultParametersValues() {
```

Comment:
All plugins copy-pasting it and no reason to have it private
https://github.com/jenkinsci/github-pullrequest-plugin/blob/6e6e457352d094b916b090972464ad401f1b2a46/src/main/java/org/jenkinsci/plugins/github/pullrequest/trigger/JobRunnerForCause.java#L123-L150

https://github.com/jenkinsci/dockerhub-notification-plugin/blob/6bd035ecfa2b8e46b7ef475c088668636e89b017/src/main/java/org/jenkinsci/plugins/dockerhub/notification/DockerHubWebHook.java#L198

https://github.com/jenkinsci/gerrit-trigger-plugin/blob/2224079b54947ddf8e90284e3ec9a8d0866d5b78/src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/EventListener.java#L259

https://github.com/search?q=user%3Ajenkinsci++%22getDefaultParametersValues%22&ref=searchresults&type=Code&utf8=%E2%9C%93